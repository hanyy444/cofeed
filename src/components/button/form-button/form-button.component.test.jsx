import { render, screen } from "@testing-library/react";
import FormButton from "./form-button.component";

describe('Test form-button component', () => { 
    it('Should render without crashing', () => {
        expect.assertions(2)
        render(<FormButton/>)
        const formButton = screen.getByTestId('form-button')
        expect(formButton).toBeInTheDocument()
        expect(formButton).toMatchSnapshot()
    })
 })