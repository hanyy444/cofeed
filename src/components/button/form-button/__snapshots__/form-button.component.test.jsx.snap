// Vitest Snapshot v1

exports[`Test form-button component > Should render without crashing 1`] = `
<button
  class="form-button "
  data-testid="form-button"
  id="form-button"
/>
`;
